AdblockPlusSafariExtension/AdblockPlusSafariExtension.xcconfig

```diff
@@ -0,0 +1,16 @@
+//
+//  AdblockPlusSafariExtension.xcconfig
+//  AdblockPlusSafari
+//
+//  Created by Pavel Zdenek on 16/02/2017.
+//  Copyright © 2017 Eyeo GmbH. All rights reserved.
+//
+
+PRODUCT_DISPLAY_NAME = Adblock Plus
+PRODUCT_BUNDLE_IDENTIFIER = org.adblockplus.AdblockPlusSafari.AdblockPlusSafariExtension
+CODE_SIGN_ENTITLEMENTS = AdblockPlusSafari/AdblockPlusSafariExtension.entitlements
+// Eyeo Development
+DEVELOPMENT_TEAM = GRYYZR985A
+CODE_SIGN_IDENTITY = iPhone Distribution: Eyeo GmbH (GRYYZR985A)
+// This replaces PROVISIONING_PROFILE, no UUID
+PROVISIONING_PROFILE_SPECIFIER = TODO RELEASE PROVISIONING PROFILE NAME
```

Comment:
Fill in the symbolic provisioning profile name that you have for the Appstore release credentials